### DISCUSSION TITLE
中文乱码

### DISCUSSION THREAD

**Discussion by Tukali** [G3G4X5X6/ultimate-cube#9] [Q&A] [answered]
win11,系统信息中文乱码
![image](https://user-images.githubusercontent.com/45891208/148634025-4c10261d-1ab8-41ed-811b-3010ce1a9f12.png)

**Comment by G3G4X5X6** [comment, marked as answer]
@Tukali 乱码的原因是 `ultimateshell` 默认使用了 `utf-8` 编码，而操作系统默认编码可能是GBK等，所以出现了乱码现象。因此，一个解决办法是：更改操作系统默认编码，设置 -> 时间和语言 -> 语言 & 区域 -> 管理语言设置 -> 更改系统区域设置 -> 勾选 `Beta版：.....` -> 重启或者注销重新登陆系统
> 注意：修改该配置可能造成其他中文程序（没有自适应编码的程序）的乱码，下个版本修复乱码错误，自适应操作系统编码。
